docs/2.x/migrate-piwik-2-to-3.md

```diff
@@ -1,235 +0,0 @@
----
```

Comment:
FYI: We cannot simply remove documents as they are referenced in the menu. We need then to also adjust the linking document which I did just here: https://github.com/piwik/developer-documentation/commit/7727bafd07f4dafd10e25916955de99778c8df89#diff-42780b1e896c88c6ac34a0e8cab1fcabL2 . Basically I removed it from that submenu and because there is no submenu left I have also removed the title so the category "Migration"  won't appear for Piwik 2.x at all.